apfel/meta.yaml

```diff
@@ -5,19 +5,19 @@ package:
 source:
     git_url: https://github.com/scarrazza/apfel
     git_rev: master
-    patches:
-      - disable_pyrap.patch
+
 requirements:
     build:
         - {{compiler('cxx')}}
         - {{compiler('fortran')}}
+        - cmake >=3.16
     host:
-        - lhapdf
+        - lhapdf==6.5.4
```

Comment:
Given that updating lhapdf made apfel fail in nnpdf I'd rather have it fixed.

Note that apfel will (soon?) become an optional dependency so it shouldn't matter.
And since I'm the one maintaining the conda lhapdf package I just need to remember to rebuild the packages here :P